scss/utilities/_stretched-link.scss

```diff
@@ -11,5 +11,8 @@
     left: 0;
     z-index: 1;
     content: "";
+    // IE10 bugfix (see https://stackoverflow.com/questions/16947967/ie10-hover-pseudo-class-doesnt-work-without-background-color)
+    // rgba(0, 0, 0, 0) is converted to transparent by sass, so we use rgba(1, 1, 1, 0)
+    background-color: rgba(1, 1, 1, 0);
```

Comment:
We need to verify clean-css doesn't do this either. See https://github.com/twbs/bootstrap/pull/26400#issuecomment-385379463

Reply:
I just checked this with the latest Bootstrap build, it stayed `rgba(1,1,1,0)`

Reply:
No, I meant `rgba(0, 0, 0, 0)` -> `transparent`, because it's definitely one of the optimization clean-css applies. So, if we need this to be kept in more places, we'd need to verify and either disable the optimization or find another workaround.

Reply:
I merged with `v4-dev` today and it looks like `rgba(0, 0, 0, 0)` is not converted to `transparent` anymore (minified and not minified). Shall I change it to `rgba(0, 0, 0, 0)` then?

Reply:
Here you go:
```
diff --git a/dist/css/bootstrap.css b/dist/css/bootstrap.css
index 8622f280e..53dbac68e 100644
--- a/dist/css/bootstrap.css
+++ b/dist/css/bootstrap.css
@@ -7424,6 +7424,17 @@ button.bg-dark:focus {
   max-height: 100% !important;
 }
 
+.stretched-link::after {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+  content: "";
+  background-color: rgba(0, 0, 0, 0);
+}
+
 .m-0 {
   margin: 0 !important;
 }
```